streamer/src/recvmmsg.rs

```diff
@@ -34,76 +41,78 @@ pub fn recv_mmsg(socket: &UdpSocket, packets: &mut [Packet]) -> io::Result<(usiz
     Ok((total_size, i))
 }
 
+#[cfg(target_os = "linux")]
+fn cast_socket_addr(addr: &sockaddr_storage, hdr: &mmsghdr) -> Option<InetAddr> {
+    use libc::{sa_family_t, sockaddr_in, sockaddr_in6};
+    const SOCKADDR_IN_SIZE: usize = std::mem::size_of::<sockaddr_in>();
+    const SOCKADDR_IN6_SIZE: usize = std::mem::size_of::<sockaddr_in6>();
+    if addr.ss_family == AF_INET as sa_family_t {
+        if hdr.msg_hdr.msg_namelen == SOCKADDR_IN_SIZE as socklen_t {
```

Comment:
combine the ss_family and msg_namelen checks with &&

Reply:
done